inst/examples/colourInput/app.R

```diff
@@ -116,6 +117,39 @@ shinyApp(
 
     div(
       class = "section",
+      div(class = "title", "Allow alpha"),
```

Comment:
I may completely change the way these boxes are rendered because it already is kinda broken sometimes depending on your screen width, so no worries about this for now